apps/website/pages/storybook.html

```diff
@@ -0,0 +1,42 @@
+---
+title: Origami Storybook
+description: Storybook for Origami components and services
+permalink: /storybook/
+layout: o-layout-landing
+---
+
+<div class="o-layout__hero o-layout__hero--muted o-layout-typography">
+	<h1>{{page.title}}</h1>
+	<p>
+		Storybook is allowing us to make the most of a large ecosystem of tools to
```

Comment:
Instead of sell storybook here I think it would be more helpful to concisely explain to our users what Storybook is. Maybe something like this?

 ```suggestion
		Interactive component demos and documentation.
```